website/blog/2022-06-16-resources-migrating-your-react-native-library-to-the-new-architecture.md

```diff
@@ -0,0 +1,112 @@
+---
+title: Resources - Migrating Your React Native Library to the New Architecture
+author: 'cipolleschi'
+tags: [announcement]
+date: 2022-06-16
+---
+
+**tl; dr**: We are working on improving the resources supporting the react Native New Architecture. We already released a repository to help migrating your app ([RNNewArchitectureApp](https://github.com/react-native-community/RNNewArchitectureApp)) and one for your libraries ([RNNewArchitectureLibraries](https://github.com/react-native-community/RNNewArchitectureLibraries)). We are also revamping the [New Architecture guide](https://github.com/facebook/react-native-website/pull/3037) on the Website and we created a [GitHub Working Group](https://github.com/reactwg/react-native-new-architecture/discussions) to answer questions related to the new Architecture.
+
```

Comment:
```suggestion

<!--truncate-->

```
We should add truncation marker somewhere, since the post is quite long. Not sure if it's an ideal spot, maybe we can leave also a bit of Intro.